src/Components/SideBar/SideBar.jsx

```diff
@@ -1,44 +1,36 @@
+import { useState } from "react";
 import styles from "./SideBar.module.scss";
-
+import sidebarData from "./sidebarData.json";
```

Comment:
Use the fetch hook made by @JDeepD. Ask him how to use it, understand how it was written, and import the data using the hook (ie. in essence, using fetch API)